lib/web_ui/test/html/text_test.dart

```diff
@@ -94,6 +94,32 @@ Future<void> testMain() async {
     }
   });
 
+  test('Can disable rounding hack', () {
+    if (!ParagraphBuilder.shouldDisableRoundingHack) {
+      ParagraphBuilder.setDisableRoundingHack(true);
+      addTearDown(() => ParagraphBuilder.setDisableRoundingHack(false));
+    }
+    assert(ParagraphBuilder.shouldDisableRoundingHack);
+    const double fontSize = 1;
```

Comment:
I noticed that the fontSize is `floor`ed in the css. Do browsers not support non-integer font sizes?